spec/mongo/libmongocrypt/binary_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'mongo'
+require 'support/lite_constraints'
+
+RSpec.configure do |config|
+  config.extend(LiteConstraints)
+end
+
+describe Mongo::Libmongocrypt::Binary do
+  require_libmongocrypt
+
+  let(:bytes) { [104, 101, 108, 108, 111] }
+  let(:binary) { described_class.new(bytes) }
+
+  after(:each) do
+    binary.close if binary
```

Comment:
This instantiates the binary, therefore the condition doesn't do anything meaningful.

Reply:
🤦‍♀ right